src/map/status.cpp

```diff
@@ -2724,8 +2724,11 @@ void status_calc_misc(struct block_list *bl, struct status_data *status, int32 l
 		status->flee2 = 0;
 
 	if (status->batk) {
-		int32 temp = status->batk + status_base_atk(bl, status, level);
-		status->batk = cap_value(temp, 0, USHRT_MAX);
+		status->batk += status_base_atk(bl, status, level);
+#ifndef RENEWAL
+		if (bl->type != BL_PC)
+			status->batk = cap_value(status->batk, SHRT_MIN, SHRT_MAX);
```

Comment:
Please add a comment about this being capped for non-players on pre-re on each one of the caps, even if look obvious someone might see it in the future and wonder why we did this

Reply:
I was wondering if we shouldn't just remove capping it altogether. Officially you can't reach values that high or low and if someone adds customized stuff, they probably just want the value to be higher than what short usually allows, what do you think?